src/modules/capabilities.json

```diff
@@ -0,0 +1,502 @@
+                                "supported": true,
+                                "available": false,
+                                "permitted": false,
+                                "granted": false,
+                                "details": ["unavailable"]
+                            },
+                            {
+                                "capability": "xrn:firebolt:capability:protocol:moca",
+                                "role": "use",
+                                "supported": true,
+                                "available": false,
+                                "permitted": false,
+                                "granted": false,
+                                "details": ["disabled","unavailable"]
+                            },
+                            {
+                                "capability": "xrn:firebolt:capability:wifi:scan",
+                                "role": "use",
+                                "supported": true,
+                                "available": true,
+                                "permitted": false,
+                                "granted": false,
+                                "details": ["unpermitted"]
+                            },
+                            {
+                                "capability": "xrn:firebolt:capability:localization:postalCode",
+                                "role": "use",
+                                "supported": true,
+                                "available": true,
+                                "permitted": true,
+                                "granted": false,
+                                "details": ["ungranted"]
+                            },
+                            {
+                                "capability": "xrn:firebolt:capability:localization:postalCode",
+                                "role": "use",
+                                "supported": true,
+                                "available": true,
+                                "permitted": true,
+                                "granted": false,
+                                "details": ["ungranted"]
+                            },
+                            {
+                                "capability": "xrn:firebolt:capability:localization:locality",
+                                "role": "use",
+                                "supported": true,
+                                "available": true,
+                                "permitted": true,
+                                "granted": false,
+                                "details": ["grantDenied","ungranted"]
+                            }
+                        ]
+                    }
+                }
+            ]
+        },
+        {
+            "name": "request",
+            "summary": "Requests grants for all capability/role combinations in the roles array.",
+            "params": [
+                {
+                    "name": "roles",
```

Comment:
`Capabilities.request()` should take an array of `Permissions` objects (formerly called `Grant`), not just a list of roles.